content/docs/command-reference/exp/remove.md

```diff
@@ -85,7 +85,7 @@ $ dvc exp remove e41d5b4 split16
 $ dvc exp show --include-params=train.min_split --no-pager
 ```
 
-```table
+```dvctable
 ┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
 ┃ Experiment            ┃ Created      ┃ State  ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
```

Comment:
should we highlight the header (colorize params, metrics, etc)?

Reply:
Idk how to do that ⏳

Reply:
OK done in d866b025 (I think)